plugins.json

```
@@ -1339,5 +1339,19 @@
         "IcoPath": "https://cdn.jsdelivr.net/gh/kenty02/Flow.Launcher.Plugin.JetBrainsIDEProjects/Flow.Launcher.Plugin.JetBrainsIDEProjects/icon.png",
         "DateAdded": "2023-02-19T21:47:51Z",
         "LatestReleaseDate": "2023-03-01T00:58:12Z"
-    }
-]
\ No newline at end of file
+    },
+    {
+    "ID": "d58fbe34-bce7-11ed-afa1-0242ac120002",
+    "ActionKeyword": "-yt",
+    "Name": "Youtube Downloader",
+    "Description": "A youtube downloader",
+    "Author": "LordOfTheEel",
+    "Version": "1.0.0-alpha",
+    "Language": "python",
+    "Website": "n/a",
+    "IcoPath": "cat.png",
+    "ExecuteFileName": "main.py",
+    "UrlSouceCode": "https://github.com/DeepVoyager253/FlowLauncherYTDownload"
```

```suggestion
    "UrlSouceCode": "https://github.com/DeepVoyager253/FlowLauncherYTDownload",
```

Looks like a missing `,`